### ISSUE TITLE
Beryllium Pixel Experience Plus (A11) sound cracking issues

### ISSUE THREAD

**Issue by aghilesh17** [PixelExperience/android-issues#188]
## Build date

```
PixelExperience_Plus_beryllium-11.0-20210519-0726-OFFICIAL
```

## Expected Behavior

```
Previously I was in MiUI 12 and the sound cracking at higher volumes was not observed.
```

## Current Behavior

```
Sound cracking while media, ringtone or notification rings at volumes higher than 60/70%
```

## Possible Solution

```
May be sound related calibration or drivers.
```

## Steps to Reproduce

```
1. Play media in social networks, YouTube or any app at volume above 60%
2. Ringtones/Notifications alerts when volume set above 60%
```

/device beryllium
/version eleven_plus
[logcat[1].txt](https://github.com/PixelExperience/android-issues/files/6660242/logcat.1.txt)

**Comment by lakgarg** [comment]
Crackling sound was already fixed in the PixelExperience_beryllium-11.0-20210420-1257-OFFICIAL build in April and no such issue persists now.

**Comment by aghilesh17** [comment]
Is this hardware issue?
Or any other work around to solve this issue for my device?
